app/src/main/java/org/simple/clinic/settings/SettingsRepository.kt

```diff
@@ -0,0 +1,10 @@
+package org.simple.clinic.settings
+
+import io.reactivex.Single
+
+class SettingsRepository {
```

Comment:
The details of this class will get implemented later when integrating with the UI. Since there is no screen to invoke this at the moment, this will not have any effect on the app.